### PR TITLE
Added multiple badge page support

### PR DESCRIPTION
**Pull request by RedSparr0w**

### CODE REVIEW

**Comment by DoctorMcKay** [comment]
Please prefix global variables with `g_`.

**Comment by RedSparr0w** [comment]
have updated with prefixes

File: farmer.js [hunk 1]
```diff
@@ -88,13 +90,27 @@ function checkMinPlaytime() {
...
+			
+			if ( $('.badge_row').length/250 == Math.round($('.badge_row').length/250) ){
+						g_Page++;
+						log("Checking page "+g_Page+" for drops");
+						checkMinPlaytime(g_Page);	
+						return;
+			}
+			
+			log(""+$('.badge_row').length+" apps with badges found");
+			log("Checking app playtime...");
```
Comment (RedSparr0w) [inline farmer.js]: Moved log("Checking app playtime..."); to here so it will not be repeated each time a new page is checked.

File: farmer.js [hunk 2]
```diff
@@ -88,13 +90,27 @@ function checkMinPlaytime() {
 			g_Jar.setCookie(cookie, 'https://steamcommunity.com');
 		});
 		
-		request("https://steamcommunity.com/my/badges/", function(err, response, body) {
+		request("https://steamcommunity.com/my/badges/?p="+g_Page, function(err, response, body) {
 			if(err || response.statusCode != 200) {
 				log("Couldn't request badge page: " + (err || "HTTP error " + response.statusCode) + ". Retrying in 10 seconds...");
-				setTimeout(checkMinPlaytime, 10000);
+				setTimeout(checkMinPlaytime(g_Page), 10000);
```
Comment (DoctorMcKay) [inline farmer.js]: This won't work.

File: farmer.js [hunk 3]
```diff
@@ -72,15 +75,14 @@ client.on('loggedOn', function() {
...
+	checkMinPlaytime(g_Page);
 });
 
 client.on('error', function(e) {
 	log("Error: " + e);
 });
 
-function checkMinPlaytime() {
-	log("Checking app playtime...");
+function checkMinPlaytime(g_Page) {
```
Comment (DoctorMcKay) [inline farmer.js]: Don't name a function argument the same thing as a global variable.

**Comment by RedSparr0w** [comment]
have updated the code, removed a few things, updated a few things.

**Comment by RedSparr0w** [comment]
Completely changed the code again, as load times were taking too long.

Re-added `g_Page` global variable.

Once the first/current page is finished Idling the next page will start.

Only idles 1 page at a time but better than the long load times.

**Comment by DoctorMcKay** [comment]
This is tested and works?

**Comment by RedSparr0w** [comment]
yup sure does. (although i only have 2 pages of apps)

all it does is `++g_Page;` variable after current page has 0 drops 
then basically re runs the script from `checkMinPlaytime();`

script will always start from page 1 in case user has new apps etc.

only problem i can see is if the user has exactly 250 apps on the last page it will keep checking that page over and overdue to if `g_Page` > your last page it will display the [last page of your badges](https://steamcommunity.com/my/badges/?p=1234)

**Comment by DoctorMcKay** [comment]
Alright, thanks. I'll push out an update before long.

**Pull request merged**